src/Storefront/Page/Product/Review/ProductReviewsLoadedCompatibilitySubscriber.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Page\Product\Review;
+
+use Shopware\Core\Content\Product\SalesChannel\Review\ProductReviewsLoadedEvent as CoreProductReviewsLoadedEvent;
+use Shopware\Core\Framework\Feature;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Storefront\Framework\Page\StorefrontSearchResult;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * @internal
+ *
+ * @deprecated tag:v6.6.0 will be re<p></p>moved without replacement, since \Shopware\Storefront\Page\Product\Review\ProductReviewsLoadedEvent will be removed
+ */
+#[Package('storefront')]
+class ProductReviewsLoadedCompatibilitySubscriber implements EventSubscriberInterface
+{
+    /**
+     * @internal
+     */
+    public function __construct(private readonly EventDispatcher $eventDispatcher)
+    {
+    }
+
+    public static function getSubscribedEvents()
+    {
+        Feature::triggerDeprecationOrThrow(
```

Comment:
we cannot trigger deprecation warnings here, as this method will be called on every kernel boot, same for the the listener, we cannot throw deprecation warnings whenever the not deprecated event is thrown, with the `reason:remove-subscriber` keyword in the deprecation annotation of this class you can silence the phpstan errors that all methods need to throw deprecation messages